doc/source/user_guide/text.rst

```diff
@@ -267,14 +267,13 @@ i.e., from the end of the string to the beginning of the string:
    s3
    s3.str.replace("^.a|dog", "XX-XX ", case=False, regex=True)
 
-.. warning::
+Single character pattern with ``regex=True`` will also be treated as regular expressions:
```

Comment:
this line seems weird without the context that it didn't use to be the case

Reply:
Fair point; will add a `.. versionchanged 2.0` here